test/parallel/test-tls-alert.js

```diff
@@ -32,15 +31,17 @@ if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-const tls = require('tls');
 
+const assert = require('assert');
+const { spawn } = require('child_process');
```

Comment:
At the moment, the test writing guide says:
> The require statements are sorted in ASCII order (digits, upper case, _, lower case).

It's not clear to me if this refers to the variable names or the module names. Thoughts?